_posts/2022/08/2022-08-31-new-while-dataization-result.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+date: 2022-08-31
+title: "New dataization result of While object"
+author: Graur
+---
+
+As in all other programming languages, an object [while](https://github.com/objectionary/eo/blob/master/eo-runtime/src/main/eo/org/eolang/bool.eo#L55) 
```

Comment:
@Graur sooner or later `master` will move forward and your link with the line 55 will become wrong. That's why it's better to refer to files in some Git tag, not in master branch.

Reply:
@yegor256 Good point, thanks!